modules/custom/entity_access_by_field/entity_access_by_field.install

```diff
@@ -47,7 +47,7 @@ function entity_access_by_field_update_8003() {
       foreach ($data['bundles'] as $bundle) {
         $config_factory
           ->getEditable("field.field.{$entity_type_id}.{$bundle}.{$field_name}")
-          ->set('default_value', NULL)
+          ->set('default_value', [])
```

Comment:
The update hook you're changing here was [added 4 years ago](https://github.com/goalgorilla/open_social/commit/f71cc2eb7bb9c5875a6ec10e0090d932313fa627) That's not going to run on any platform. Please add a new update hook instead.

Reply:
I know that this update hook was added 4 years ago, I did changes there with the thought that someone might copy it in the future and repeat the same mistake 😔
But yeah we can add a new hook in this module, that will set the correct value.

Reply:
Ah that's something I hadn't considered! Definitely a good point! 

One thing we can also do if this update hook is more than 2 major versions old is remove the update hook and implement `hook_update_last_removed` instead to make sure people no longer copy it :D (We should wait with deleting 9.x update hooks until 11.0 though, not sure where this one falls in)